bee-protocol/src/worker/requester/transaction.rs

```diff
@@ -30,21 +31,26 @@ use std::time::{Duration, Instant};
 
 const RETRY_INTERVAL_SECS: u64 = 5;
 
+pub(crate) type RequestedTransactions = DashMap<Hash, (MilestoneIndex, Instant)>;
```

Comment:
I would encourage using a newtype with a `Deref` instead of a type alias. The resource uses `TypeId`, and so uniqueness is based on the type signature. Since type aliases don't actually create a new type, they just get the `TypeId` of the underlying type. Should someone ever add a similar resource with the same signature, they would conflict.